jujugui/static/gui/src/test/test_app.js

```diff
@@ -1,2226 +0,0 @@
-/*
```

Comment:
We'll probably want to revisit this file to write the tests for the init, but it being here causes issues when trying to get a running suite.

Reply:
I wonder if we could skip these tests instead of removing them, then we can remove the parts as we port them over to keep track of what has been done. Thoughts?

Reply:
I figured that because we would be rewriting them and only making sure we cover the same functionality the code itself probably wouldn't be very helpful whereas having it in here could cause some issues with dependency loading that we don't need to deal with